packages/block-library/src/navigation/index.php

```diff
@@ -410,6 +410,25 @@ function( $block ) {
 	return array_values( $filtered );
 }
 
+/**
+ * Returns true if the navigation block contains a nested navigation block.
+ *
+ * @param array $parsed_blocks the parsed blocks to be normalized.
+ * @return bool true if the navigation block contains a nested navigation block.
+ */
+function block_core_navigation_has_nested_core_navigation( $parsed_blocks ) {
```

Comment:
```suggestion
function block_core_navigation_block_contains_core_navigation( $parsed_blocks ) {
```

Nit: I think we can just name this to explain that it's testing a set of blocks for the presence of a core navigation block. The argument is "a collection of blocks" not "a core navigation block".

It's not "does this core navigation block contain core navigation blocks".